README.md

```diff
@@ -16,50 +16,61 @@
+that we use regularly.
 
-Command | Purpose
----|---
-`ldmx cmake ..` | Configure the ldmx-sw build
-`ldmx make` | Compile/build ldmx-sw
-`ldmx make install` | Install ldmx-sw
-`ldmx compile` | Configure and compile ldmx-sw
-`ldmx fire config.py` | Use ldmx-sw application and processors with input python configuration
-`ldmx recompFire config.py` | Recompile and run fire on a config file
-`ldmx python3 analysis.py` | Run python-based analysis
-`ldmx ./bin/mg5_aMC` | Run MadGraph5 inside (ubuntu-based) container
+> [!WARNING]
+> If you are on Windows, make sure to install these tools _inside_ WSL where `docker`
+> will be run and ldmx-sw will be developed. Since WSL is often a virtual Ubuntu machine,
+> following the instructions for Ubuntu or Linux can be appropriate.
 
-### Other Container Configuration Commands
+- Make sure `git` is installed.
+  - `git` is a very common tool used by software developers and so it may already be available.
+  - (on MacOS systems) Make sure `git lfs` is installed. (Test: `git lfs` prints out a help message instead of an error about `lfs` not being found.) The default installation of `git` that is included with Apple's developer tools does not include `git lfs` which is required by acts to download and unpack one of its own submodules. [GitHub has a nice tutorial](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=mac) on how to install `git lfs` on MacOS.
+- [Install `just`](https://just.systems/man/en/chapter_5.html)
+  - This tool is not required but it is highly encouraged. The recipes we share via the [justfile](justfile) can be run without `just` but are longer to type.
 
-The environment script defines several other shell commands to help configure and debug the container environment.
+Then, with these additional tools, developers can clone the repository and start development.
+```
+git clone --recursive git@github.com:LDMX-Software/ldmx-sw.git
+```
 
-- `ldmx list repo` : List the container tags that you could use with the input repository: `dev`, `pro`, or `local`
-- `ldmx use repo tag` : Setup the environment for the container 'ldmx/repo:tag' and pull down the newest version if the repo is remote
-- `ldmx config` : Print out how the container environment is currently configured
-- `ldmx clean all` : Reset environment to a blank state
+> [!NOTE]
+> You need to [setup an SSH-key with your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) on the computer you are using.
 
-Use `ldmx help` for a full listing of these commands.
-If we don't define a command outside of the container,
-then the command is given to the container to run inside the current directory.
+```
+cd ldmx-sw
+just # no arguments prints out the possible options
+just init # initialize a new development environment (once per clone)
+just configure build test # configure ldmx-sw, build it, then test it
```

Comment:
I'll add it here for a lack of better point to comment on, I'd like to have autocomplete back.

Reply:
Reasonable feature request, it is close to available and I can include links and information in the README. `just` tab complete is handled at install time (either via the package manager or by the user manually). Tab complete of a `just` alias requires me to update the `ldmx-env.sh` script.

https://github.com/LDMX-Software/ldmx-sw/pull/1380#issuecomment-2322164988